auction_keeper/main.py

```diff
@@ -403,13 +435,21 @@ def check_all_auctions(self):
 
         self.logger.debug(f"Checked {self.strategy.kicks()} auctions in {(datetime.now() - started).seconds} seconds")
 
+    def check_for_bids(self):
```

Comment:
why is this function declared twice?

Reply:
Bad merge.  Resolved; will retest before pushing.